docs/introduction/installation.md

```diff
@@ -0,0 +1,120 @@
+- [aframe-registry](https://glitch.com/~aframe-registry) - Showcase of various components.
+- [aframe-vaporwave](https://glitch.com/~aframe-vaporwave) - Retro-futuristic scene.
+- [networked-aframe](https://glitch.com/~networked-aframe) - Multiuser.
+
+### Other Code Editors
+
+Below are a couple other A-Frame starter kits on other browser-based code
+editors. Both support remixing or forking:
+
+- [Mozilla Thimble &mdash; A-Frame](https://thimble.mozilla.org/en-US/user/ngokevin/864056)
+- [CodePen &mdash; A-Frame](https://codepen.io/mozvr/pen/BjygdO)
+
+## Local Development
+
+### Use a Local Server
+
+For the options below, we should develop projects using a local server so that
+files are properly served. Options of local servers include:
+
+- Downloading the [Mongoose](https://www.cesanta.com/products/binary) application
+  and open it from the same directory as your HTML file.
+- Running `python -m SimpleHTTPServer` in a terminal in the same directory as your
+  HTML file.
+- Running `npm install -g live-server && live-server` in a terminal in the same
+  directory as your HTML file.
+
+### Download the Boilerplate on GitHub
+
+[ghpages]: https://pages.github.com/
+
+The boilerplate contains:
+
+- A simple HTML file that links to the [current version of A-Frame](#builds-prod)
+- An optional local development server
+- An easy deployment workflow for [GitHub Pages][ghpages] to share with the world
+
+We can grab the boilerplate in one of two ways:
+
+<a class="btn btn-download" href="https://github.com/aframevr/aframe-boilerplate/">Fork on GitHub</a>
+
+<a class="btn btn-download" href="https://github.com/aframevr/aframe-boilerplate/archive/master.zip" download="aframe-boilerplate.zip">Download .ZIP<span></span></a>
+
+### Include the JS Build
+
+To include A-Frame into an HTML file, we drop a `<script>` tag pointing to the
+CDN build:
+
+```html
+<head>
+  <script src="https://aframe.io/releases/0.5.0/aframe.min.js"></script>
+</head>
+```
+
+If we want to serve it ourselves, we can download the JS build:
+
+<a id="builds-prod" class="btn btn-download" href="https://aframe.io/releases/0.5.0/aframe.min.js" download>Production Version <span>0.5.0</span></a> <em class="install-note">Minified</em>
+<a id="builds-dev" class="btn btn-download" href="https://aframe.io/releases/0.5.0/aframe.js" download>Development Version <span>0.5.0</span></a> <em class="install-note">Uncompressed with Source Maps</em>
+
+### Install from npm
+
+We can also install A-Frame through npm:
+
+```bash
+$ npm install aframe
+```
+
+Then we can bundle A-Frame into our application. For example, with Browserify
+or Webpack:
+
+```js
+require('aframe');
+```
+
+[angle]: https://www.npmjs.com/package/angle
+
+If you use npm, you can use [`angle`][angle], a command line interface for
+A-Frame.  `angle` can initialize a scene template with a single command:
+
+```sh
+npm install -g angle && angle initscene
```

Comment:
I think this post could benefit from some mention of hosting assets. At minimum, link to the FAQ entry. Perhaps also call out differences between the different options: Glitch has asset-hosting built in, Codepen does not, etc.